src/components/ChatList/ChannelListHeaderItem.tsx

```diff
@@ -28,12 +28,12 @@ const ChannelListHeaderItem = (props: ChannelListTabItemProps) => {
       alignItems: 'center',
       backgroundColor: COLORS.white,
       borderBottomWidth: 2,
-      borderBottomColor: type === SIGNED ? COLORS.blueOnboarding : COLORS.anon_primary
+      borderBottomColor: type === SIGNED ? COLORS.signed_primary : COLORS.anon_primary
     },
     name: {
       fontFamily: fonts.inter[500],
       fontSize: normalizeFontSize(14),
-      color: type === SIGNED ? COLORS.blueOnboarding : COLORS.anon_primary
+      color: type === SIGNED ? COLORS.signed_primary : COLORS.anon_primary
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color constants have been updated from `COLORS.blueOnboarding` to `COLORS.signed_primary`. This change seems to be consistent with the PR summary and should improve the visual consistency of the application. However, ensure that this new color constant is defined in your `COLORS` object and it's visually tested across different screens for consistency.
```diff
-      borderBottomColor: type === SIGNED ? COLORS.blueOnboarding : COLORS.anon_primary
+      borderBottomColor: type === SIGNED ? COLORS.signed_primary : COLORS.anon_primary

-      color: type === SIGNED ? COLORS.blueOnboarding : COLORS.anon_primary
+      color: type === SIGNED ? COLORS.signed_primary : COLORS.anon_primary
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->